### DISCUSSION TITLE
@formily/element 的 esm build 没有去掉 import './style'

### DISCUSSION THREAD

**Discussion by Amorites** [alibaba/formily#2328] [Q&A 2.x]
导致构建的时候去加载 scss，里面有 ~element-ui 的路径无法被正确解析

**Comment by muuyao** [comment]
本身就是要去加载的
~element-ui 的路径无法被正确解析是不是用的 vite ？
如果是的话，把 ~element-ui alias 到 element-ui 就可以了
